scripts/serve-local.sh

```diff
@@ -31,4 +31,5 @@ docker run \
     --env "LOAD_DATA=$LOAD_DATA" \
     --env "GUNICORN_CMD_ARGS=--reload  --reload-engine=poll" \
     --env "JANIS_URL=http://localhost:3000" \
+    --env "STYLEGUIDE_URL=https://cityofaustin.github.io/digital-services-style-guide" \
```

Comment:
can you make sure to add this env var to heroku if you haven't already so we don't forget and break the build?